package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@dynamods/notifications-center",
-  "version": "0.0.11",
```

Comment:
Oh, dont forget to update package-lock.json with these changes

Reply:
done, thanks for reminding.